manuscript/portal_iterative_forecasting_ms.Rmd

```diff
@@ -41,32 +41,32 @@ Key-words: forecasting, prediction, mammals, iterative forecasting, Portal Proje
+To ensure that software pipelines continue to run automatically as software dependencies change, a key component of "continuous analysis" is the use of a reproducible computational environment [@beaulieu2017]. We followed Beaulieu and Greene's [-@beaulieu2017] recommendation to use software containers. Software containers are standalone packages that contain copies of everything needed to run a given piece of software, including the operating system [@boettiger2015]. Once created, a software container is basically a time capsule, containing all the software dependencies in the exact state used to develop and run the software [@boettiger2015]. If those dependencies change (or disappear) in the wider world, they still exist, unchanged, in the container. We use an existing platform, Docker [@merkel2014], to store an exact image of our complete software environment by adding our project specific code to a container created by the Rocker project, which is a Docker image with many important R packages [i.e., the tidyverse packages; @tidyverse] pre-installed  [@boettiger2017]. We implemented this system because we experienced issues with external dependencies breaking our pipeline when the `tscount` package [@liboschik2015], was temporarily removed from CRAN (the R package repository) and could not be installed in the usual way. In combination, the automated running of the pipeline (continuous integration) and the guarantee it will not stop working unexpectedly due to software dependencies (via a software container) allows continuous analysis to serve as the glue that connects all stages of the forecasting pipeline.
```

Comment:
I find the "We implemented..." sentence to be a little weird. I think it would make more sense to explicitly make the `tscount` issue an example. Something like "...external dependencies break out pipeline (e.g., when the `tscount` package..."
Not imperative, but that would read a lot better IMO